pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,36 +1,36 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
```

Comment:
@weshaggard Is there any concern in publishing a package from here that is pre-release. We can choose not to publish a stable UWP package from 1.1.0 branch, right?